tools/CAT/src/pages/editPage/index.tsx

```diff
@@ -29,6 +29,11 @@ import {
 import { fetchCategoriesData } from '@/app/features/categories/categoryThunks';
 import { showErrorNotif } from '@/providers/notify';
 
+enum TemplateType {
+	DEFAULT = 0,
+	WITH_TITLE = 1,
+}
+
```

Comment:
Duplicate code, let's use enum from previous comment

Reply:
OHH we have two identical files, one with an uppercase letter and one with a lowercase letter